src/modal/modal-window.ts

```diff
@@ -62,7 +63,17 @@ export class NgbModalWindow implements OnInit,
 
   ngAfterViewInit() {
     if (!this._elRef.nativeElement.contains(document.activeElement)) {
-      this._elRef.nativeElement['focus'].apply(this._elRef.nativeElement, []);
+      let elementToFocus = this._elRef.nativeElement.querySelector(`[ngbAutofocus]`) as HTMLElement;
+      if (!elementToFocus) {
```

Comment:
suggestion: Can you please merge this two level if statement, it looks maybe confusing (at least form me)